### PR TITLE
fervojo:0.1.0

### PR DESCRIPTION
**Pull request by leiserfg**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: 
Railroads are useful to explain state machines like in regexes or grammars.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/typst-railroad/0.1.0/lib.typ [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#let render(
+    railroad,
+    css: rr-wasm.default_css(),
+    ..args
+) = {
+    image.decode(rr-wasm.railroad(_as_bytes(railroad), _as_bytes(css)), format: "svg")
+}
+
+
+#let default_css() = {
```
Comment (elegaanz) [inline packages/preview/typst-railroad/0.1.0/lib.typ:28]: kebab-case (i.e `default-css`) is a more common convention in the Typst ecosystem (but snake_case is allowed, if you prefer to keep it).

File: packages/preview/typst-railroad/0.1.0/typst.toml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[package]
+name = "typst-railroad"
```
Comment (elegaanz) [inline packages/preview/typst-railroad/0.1.0/typst.toml:2]: Package names should not include "Typst", and should not be the most obvious/canonical name for what the package does. Can you please propose another name?

File: packages/preview/typst-railroad/0.1.0/examples/simple.typ [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#import "../lib.typ": *
```
Comment (elegaanz) [inline packages/preview/typst-railroad/0.1.0/examples/simple.typ:1]: Ideally this should use the package specification, to make it easier to get started from the example.

Comment (JamesxX) [reply packages/preview/typst-railroad/0.1.0/examples/simple.typ:1]: Is this a new requirement? Star imports are fairly idiomatic

File: packages/preview/typst-railroad/0.1.0/lib.typ (hunk 1 above)
Comment (leiserfg) [reply packages/preview/typst-railroad/0.1.0/lib.typ:28]: 👍

File: packages/preview/typst-railroad/0.1.0/examples/simple.typ (hunk 3 above)
Comment (elegaanz) [reply packages/preview/typst-railroad/0.1.0/examples/simple.typ:1]: What I meant by "package specification" was to use `#import "@preview/typst-railroad:0.1.0": *` (but with the new package name once you will have decided on it). Sorry if it was not clear.

Comment (leiserfg) [reply packages/preview/typst-railroad/0.1.0/examples/simple.typ:1]: I think @elegaanz  means it should be like `#import @preview/....`, I was testing it locally, and using it that way does not work, I will change it for clarity.

File: packages/preview/typst-railroad/0.1.0/typst.toml (hunk 2 above)
Comment (elegaanz) [reply packages/preview/typst-railroad/0.1.0/typst.toml]: Sorry, but this is still too descriptive of what the package does. Can you add a prefix/suffix, or an adjective that makes the name more unique, or make a pun with "railroad"/"train"/etc. please?

Comment (leiserfg) [reply packages/preview/typst-railroad/0.1.0/typst.toml]: Can I use the name translated to esperanto?
`fervojo`

Comment (elegaanz) [reply packages/preview/typst-railroad/0.1.0/typst.toml]: Yes, that would work.

**Comment by elegaanz** [comment]
Thank you !

**Pull request merged**